zstacklib/zstacklib/test/utils/http_test_client.py

```
@@ -0,0 +1,484 @@
+        self.send_header('Content-Type', 'application/json')
+        self.end_headers()
+
+        callback_uri = (headers.get(CALLBACK_URI)
+                        or headers.get('Callbackurl')
+                        or entry.callback_uri)
+
+        def _run():
+            cb_headers = {TASK_UUID: task_uuid}
+            try:
+                content = entry.func(entity)
+            except Exception as ex:
```

Ruff (0.15.9)

[warning] Do not catch blind exception: `Exception`

(BLE001)